lib/src/textLabel/textLabel.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="text-label" [ngClass]="className" id="id">
+    <ng-container *ngTemplateOutlet="isString(label) ? defaultLabel : label"></ng-container>
+    <ng-container *ngTemplateOutlet="isString(value) ? defaultValue : value"></ng-container>
```

Comment:
Don't you think we should replace string rendering with `ng-content`?

Reply:
`*ngTemplateOutlet` gives us some more power of showing the default template if none is provided. You cannot conditionally render the content within `ng-content`. You have to show the content that is received from the parent with no means to make decisions based on the content, which is what we want to achieve now.

Reply:
I understand that we can achieve conditional ng-content with *ngTemplateOutlet and from your comment above i can see another different between the two. But I am a little bit skeptical about adoption/implementation/usage though as far as users are concern . Do you thing the users are familiar with *ngTemplateOutlet  and it usage ?